src/app/contextBroker/orionRestServices.cpp

```diff
@@ -401,7 +401,7 @@ static RestService badVerbV[] =
   { BatchQueryRequest,                             3, { "v2", "op", "query"                                                            }, badVerbPostOnly           },
   { BatchUpdateRequest,                            3, { "v2", "op", "update"                                                           }, badVerbPostOnly           },
   { RegistrationRequest,                           3, { "v2", "registrations", "*"                                                     }, badVerbGetDeleteOnly      },
-  { RegistrationsRequest,                          2, { "v2", "registrations"                                                          }, badVerbPostOnly           },
+  { RegistrationsRequest,                          2, { "v2", "registrations"                                                          }, badVerbGetPostOnly        },
```

Comment:
Should the bugfix be mentioned in CNR (without referrencing issue number)? Or is too minor?

Reply:
I'd say it's very minor, but I have no problem with adding it to the change log.

Reply:
Ok. Add it, pls.

Reply:
Fixed in cb6054704